### ISSUE TITLE
Q2 Idea: Quick Lesson videos

### ISSUE THREAD

**Issue by joethreepwood** [PostHog/meta#188]
This is mainly me parking an issue for discussion in our upcoming Q2 planning, based on conversations @andyvan-ph and I have had around improving docs and support processes in tandem next quarter. 

## Problem
We see a lot of very basic support tickets come through which follow a very similar premise: "_How do I..._". 

Example questions recently: 
- Is it possible to create a cohort based on application errors like this "invalid token"?
- How do you actually change this report to only include the first pageview in a session?
- Is there a way to tag individual events w/ extra metadata somehow?
- How do I copy dashboards to a new project?

We solve nearly all of these in one-touch, which is great. The only thing that would be better is if we solved them in _zero_ touches, or we made it faster for agents to reply. In other words: if we had more content to cover this. 

The trouble with content is that it takes a while to write, edit, publish, and disseminate. Tutorials are already crowded. Content dates and needs to be managed. 

## Proposal
A possible solution would be to create a video playlist on YouTube of 'Quick lessons', which are literally nothing more than simple Looms or recordings showing how to do these tasks. They'd be short - less than 90 seconds each. 

The first problem with video generally is that it also takes a while to make. We'd solve this by making all of the videos short, with a consistent format, and not fussing too much about the scripts or quality. We aren't asking for likes and subscribes. We aren't aiming to give all the context. 

It's nothing more than: "Here's how to setup the filter for internal users. This helps you cut your team out of analysis in one click. To do it, go to settings, go to this section, and enter your filter criteria. A good one for most teams is using the email domain. Save it, and you can apply it in one click easily. That's it!"

We can use Yarn and a Chrome extension they provided to quickly edit the videos and automatically remove sensitive data. 

We then upload these videos to YouTube in a playlist, and can feature this playlist somewhere on the site. We set up a macro in Zendesk for simple questions that we've covered here, directing users to the channel. At the same time, we increase our Youtube footprint and hopefully improve activation. 

In my head, I'd see us doing a regular process here around the content sprints. Every two weeks, when we review the content ideas, we include some of these and plan to get the videos done. I'd guess we could do 3-4 each fortnight in this way, and quickly grow the channel.

**Comment by andyvan-ph** [comment]
Couple of considerations:

- Having lots of short vids like this on our channel may look a bit rubbish. Would it be ok to publish them unlisted?
- If not, is there an alternative platform we could host the videos on?

**Comment by jtemperton** [comment]
+1 to publishing them unlisted if we use YouTube. Don't see it as a way of growing the channel as you'd only watch the videos to solve a very specific problem. But agree videos could be a good way of addressing the problem of content volume, it becoming dated, etc.

**Comment by joethreepwood** [comment]
Defer to you both on this aspect, tbh. It's not something I have a strong feeling on.

**Comment by ivanagas** [comment]
- All of your example questions seem relatively niche. I think there are some quick videos we can do, but thinking about how users (and our team) will find them is important. Searchability matters for videos too.
- It is fine to upload them normally. A minority of views come from the channel page which would be the thing impacted by "looking rubbish". Unlisted prevents them from being searched, which is where a majority of views come from. YouTube's algorithm is really good at showing people what they want to watch. 
- The nice thing about docs and tutorials over videos is that they are way easier to edit and we get more support from the community via comments and contributions to keep them updated. A significant number of our ~1 year old videos are out of date now and require a complete reshoot to fix, where we've tweaked popular tutorials and docs to keep them updated.
- Based on my experience with YouTube, tutorial videos have very little impact on growth and don't convert well, so this would be an entirely a support play. 

TLDR: I'm fine with it but it should be lower priority than written content and entirely with the goal of helping support. 

![CleanShot 2024-03-18 at 10 35 26](https://github.com/PostHog/meta/assets/34755028/73717468-09c8-4d39-9511-454f16ee4cea)

**Comment by jtemperton** [comment]
All good points, yup. This is very good Q2 planning fodder!
